mlflow/pytorch/__init__.py

```diff
@@ -1006,51 +999,30 @@ def print_auto_logged_info(r):
             print(f"tags: {tags}")
 
 
-        # Initialize our model
+        # Initialize our model.
         mnist_model = MNISTModel()
 
-        # Initialize DataLoader from MNIST Dataset
+        # Load MNIST dataset.
         train_ds = MNIST(
             os.getcwd(), train=True, download=True, transform=transforms.ToTensor()
         )
-        train_loader = DataLoader(train_ds, batch_size=32)
+        # Only take a subset of the data for faster training.
+        indices = torch.arange(32)
+        train_ds = Subset(train_ds, indices)
+        train_loader = DataLoader(train_ds, batch_size=8)
 
-        # Initialize a trainer
-        trainer = pl.Trainer(max_epochs=20, progress_bar_refresh_rate=20)
+        # Initialize a trainer.
+        trainer = L.Trainer(max_epochs=3)
 
         # Auto log all MLflow entities
         mlflow.pytorch.autolog()
 
-        # Train the model
+        # Train the model.
         with mlflow.start_run() as run:
             trainer.fit(mnist_model, train_loader)
 
-        # fetch the auto logged parameters and metrics
+        # Fetch the auto logged parameters and metrics.
         print_auto_logged_info(mlflow.get_run(run_id=run.info.run_id))
-
-    .. code-block:: text
-        :caption: Output
-
-        run_id: 42caa17b60cb489c8083900fb52506a7
```

Comment:
Why do we remove this part?

Reply:
Maintaining an output without testing is not ideal, as it could show users wrong outputs. This output is already stale. 

If we want to show sample outputs, we need to have testing around it. For example, code shown [here](https://keras.io/api/layers/convolution_layers/convolution2d/) are covered by CI.